src/ui/view/share/knowFee/knowFee.tsx

```diff
@@ -0,0 +1,32 @@
+import type { FC } from 'react'
+import classNames from 'classnames'
+import { useAppStore } from '@/ui/store'
+import { Field } from '@/ui/component/field/field'
+import './knowFee.scss'
+
+type KnowFeeProps = {
+  readonly?: boolean
+  label: string
+}
+
+export const KnowFee: FC<KnowFeeProps> = ({ readonly = false, label }) => {
+  const theme = useAppStore(store => store.theme)
+
+  const formItems = useAppStore(state => state.shareData.form)
+  const fee = formItems.find(item => item.label === 'fee')
```

Comment:
```suggestion
  const fee = useAppStore(( shareData: { form }) => state.shareData.form).find(item => item.label === 'fee')
```
Its only a suggestion, feel free to self resolve if you dont agree

Reply:
Ah I tried but it throws a lot of errors so I prefer to keep it simple if you don't mind :)

Reply:
This one can't work because `useAppStore` expects a specific selector function that returns the memoized slice property.
It the way `zustand` works in its core.